logstash-core/src/main/java/org/logstash/ackedqueue/ext/JRubyWrappedAckedQueueExt.java

```diff
@@ -48,7 +51,6 @@ public final class JRubyWrappedAckedQueueExt extends AbstractWrappedQueueExt {
     private static final long serialVersionUID = 1L;
 
     private JRubyAckedQueueExt queue;
-    private final AtomicBoolean isClosed = new AtomicBoolean();
```

Comment:
review note: for better atomicy/consistency, exposed `Queue->isClosed()` and used instead. This is a duplicate of `Queue#closed`, hidden bug that we are allowing to write to queue even after `Queue#closed` `true` but `isClosed` is still `false`.